src/views/pages/Home.vue

```diff
@@ -106,9 +126,9 @@
 
   <div id="built-exclusively-for-you">
     <div class="container mx-auto px-12 my-32">
-      <div class="grid grid-cols-8 gap-x-4">
+      <div class="flex flex-col justify-center md:flex-row">
```

Comment:
These were shifted over to one side as the right side was going to have a picture/control flow diagram. I'm not sure how you want to do it with flex cols so I'll let u fix it.

Reply:
https://prnt.sc/EKMdB4Kck8cQ

Something like this was my idea with the dashed line being the same as that divider, lmk what u think.